### PR TITLE
adding django testing documentation

### PR DESCRIPTION
**Pull request by oliver-nowak**
This PR contains documentation around django testing. It describes how to run the unit tests, how they are organized in the source tree, and points to the Django reference guide for how to write them.

### CODE REVIEW

File: TESTING_PYTHON.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+## Location of Test suites
+Tests should be written near the feature they are testing. Currently, we have test suites in the following locations:
+* group1/communication/tests
+* group1/functional_tests
+* issue_tracker/tests
+* project_router/tests
+* requirements/tests
+
+## Writing Tests
+In order to write a test for a feature, locate the test suite the represents the "area of concern". For instance, if you are working on the communication tool, you would look for the test suite in **group1/communication/tests**.
```
Comment (ghost) [inline TESTING_PYTHON.md]: Small typo here with "...locate the test suite the represents..."

**Review by ghost — COMMENTED** [review]
Looks pretty good. It might be nice to give some example code for a small unit test as well.

File: TESTING_PYTHON.md (hunk 1 above)
Comment (oliver-nowak) [reply TESTING_PYTHON.md]: nice catch, william. let me fix that.

**Comment by oliver-nowak** [comment]
@wtbaldoumas yah, i thought about adding a small example. but the reference url i link at the bottom has some nice examples, so i thought it would be redundant. what do you think ?

**Pull request merged**